### PR TITLE
README.md: Update company and copyright year

### PR DESCRIPTION
**Pull request by thenor**
Signed-off-by: Therese Nordqvist <therese.nordqvist@pelagicore.com>

### CODE REVIEW

**Comment by thenor** [comment]
@jeremiah @frznlogic Any opinions on if we can use "Pelagicore" since it is still the project name in GitHub? Or was it correct to change it to "PELUX"? It felt wrong to change it to "Luxoft", since it's the name of the company, not the project.

**Comment by frznlogic** [comment]
Personally I'm fine with it. Did you check the other files for any possible copyright info etc?

**Comment by thenor** [comment]
@frznlogic Nope. I could do a grep for it while I make this change. I was reading this and saw that old year and incorrect company name

**Comment by frznlogic** [comment]
@thenor yapp, grep for it and let's update it all imho :)

File: build/build-documentation.sh [hunk 1]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 # vagrant-cookbook
-# Copyright (C) 2015 Pelagicore AB
+# Copyright (C) 2015-2017 Pelagicore AB
+# Copyright (C) 2017-2018 Luxoft Sweden AB
+# Copyright (C) 2017-2018 Luxoft Sweden AB
```
Comment (thenor) [inline build/build-documentation.sh]: This was unnecessary. I will fix it.

File: build/clang-code-analysis.sh [hunk 2]
```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 # vagrant-cookbook
-# Copyright (C) 2015 Pelagicore AB
+# Copyright (C) 2015-2017 Pelagicore AB
+# Copyright (C) 2017-2018 Luxoft Sweden AB
+# Copyright (C) 2017-2018 Luxoft Sweden AB
```
Comment (thenor) [inline build/clang-code-analysis.sh]: Not as easy as it seemed...

**Comment by thenor** [comment]
@jeremiah So you mean that keeping "Copyright (c) 2015 Pelagicore AB" is correct also? That would mean that I don't need to make this change.

**Comment by jeremiah** [comment]
I do mean that keeping "Copyright (c) 2015 Pelagicore AB" is correct, yes. And I agree, I see no pressing reason to make this change. (There are some spots where it just says "Pelagicore" without the AB, that ought to get fixed one day, but not a huge issue.)

**Comment by thenor** [comment]
Considering the feedback from @jeremiah I will close this user story and not make this change at this moment. We will add it to the backlog for future consideration instead.

**Pull request closed without merging**